pkg/morph/client/client.go

```diff
@@ -308,20 +406,36 @@ func toStackParameter(value interface{}) (sc.Parameter, error) {
 			result.Value = int64(0)
 		}
 	default:
-		return result, fmt.Errorf("chain/client: unsupported parameter %v", value)
+		return result, neofsError{fmt.Errorf("chain/client: unsupported parameter %v", value)}
```

Comment:
is there any logic when you choose b/w `noefsError{}` and `wrapNeoFSError()` or its a mistake?

Reply:
Mistake, fixed